sunkit_image/tests/test_enhance.py

```diff
@@ -29,6 +29,12 @@ def map_test():
     return np.ones((4, 4), dtype=float)
 
 
+@pytest.mark.remote_data
+def test_wow(aia_171):
+    _ = enhance.wow(aia_171.data)
+
+
+@pytest.mark.xfail
```

Comment:
This is failing?

Reply:
Thats not a problem, we can use the CI on Github to check! I do it all the time.